src/components/angular2-modal/plugins/bootstrap/bootstrap.module.ts

```diff
@@ -1,7 +1,7 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 
-import { ModalModule, Modal as BaseModal } from '../../../../components/angular2-modal';
+import { ModalModule, Modal as BaseModal } from '../../../angular2-modal';
```

Comment:
These references used to cause issue with SystemJS and other unpredicted stuff.
I changed them to a full reference and replace them before the build anyway, each is translated to `angular2-modal`